Use mamba to spin up development environments (#2520)
ilastik/ilastik: CONTRIBUTING.md

   gh repo clone ilastik
   ```

> 1. We recommend to downloading/using Mambaforge to interact with conda.
>    Mambaforge already comes configured correctly (this step can be skipped in this case).
>    If you are using [Miniconda][miniconda] instead, please make sure to configure conda to use [strict channel priority][strict-channel-priority], and install the fast [Mamba][mamba] solver for conda environments:

Reviewer: ```suggestion
1. If you have an existing [Miniconda][miniconda] installation, you can use it for ilastik, but make sure to install [mamba][mamba] and configure conda to use [strict channel priority][strict-channel-priority]:
```

   ```
   conda config --set channel_priority strict

>    conda install --name-base -c conda-forge mamba

Reviewer: ```suggestion
   conda install --name base -c conda-forge mamba
```

   ```

1. Install [conda-build][conda-build] in order to access the [conda develop][conda-develop] command:

   ```

>    mamba install --name base -c conda-forge conda-build

Reviewer: Are we also moving to `boa` from `conda-build`?

Author: when using conda-develop it doesn't make a difference. but yeah, for actual building packages I'm using boa right now.
